virtcontainers/sandbox.go

```diff
@@ -837,7 +837,11 @@ func (s *Sandbox) Delete() error {
 	}
 
 	s.agent.cleanup(s)
-
+	if  useOldStore(s.ctx) && s.store != nil {
```

Comment:
@keloyang what's the reason of only doing this for an old store?

Reply:
useOldStore  return true only loadSandboxConfigFromOldStore return successfully,  so I add useOldStore here, but it's not necessary.